requirements.txt

```diff
@@ -0,0 +1,54 @@
+async-generator==1.10
```

Comment:
do we really need all of these packages to run your training?

Reply:
Whoops